mithril-core/src/merkle_tree.rs

```diff
@@ -0,0 +1,422 @@
+//! Creation and verification of Merkle Trees
```

Comment:
As we talked about it this file is a leftover from rebasing.  We need to remove the entire `mithril-core` folder.